src/utils/index.js

```diff
@@ -1,2 +1,5 @@
 export { postNotesHandler } from "./services";
 export { postNoteEditHandler } from "./services";
+export { postArchiveNoteHandler } from "./services";
+export { postRestoreArchiveHandler } from "./services";
+export { deleteArchiveNoteHandler } from "./services";
```

Comment:
export all this functions in a single export

Reply:
Done